discovery-provider/src/api_helpers.py

```diff
@@ -30,23 +30,16 @@ def default(self, o): # pylint: disable=E0202
 def error_response(error, error_code=500):
     return jsonify({'success': False, 'error': error}), error_code
 
-# Create a response dict with just data, signature, and timestamp
-# This response will contain a duplicate of response_entity
-def success_response_backwards_compat(response_entity=None, status=200):
```

Comment:
unfortunately i don't think we can remove this quite yet. i think while libs are upgraded, there are places that are still using the old format https://github.com/AudiusProject/audius-protocol/blob/master/identity-service/src/routes/healthCheck.js#L216

Reply:
fair enough. won't remove yet :)